h/templates/groups/create_edit.html.jinja2

```diff
@@ -11,6 +9,21 @@
         "createGroup": {
           "method": "POST",
           "url": "{{ request.route_url("api.groups") }}"
+        },
+        "updateGroup": {
+          "method": "PATCH",
+          "url": "{{ request.route_url("api.group", id=group.pubid) }}"
+        }
+      },
+      "components": {
+        "CreateEditGroupForm": {
+          "mode": "{{ mode }}",
+          "group": {
+            "pubid": "{{ group.pubid }}",
+            "name": "{{ group.name }}",
+            "description": "{{ group.description }}",
+            "link": "{{ group.link }}"
+          }
```

Comment:
I got confused initially while looking at this code because the shape of the `group` object is defined in both `groups.py` and here.

Separately, Jinja is a crap environment to define JSON structures in. I would suggest in future reworking the code so that we build up the configuration as a JSON-serializable Python dict and then the template just does `{{ config | tojson }}`. The code that generates the dict is then easy to execute in a test as well.